dev/deps/spark-deps-hadoop-3.2

```diff
@@ -79,7 +78,21 @@ hadoop-yarn-common-3.2.0.jar
 hadoop-yarn-registry-3.2.0.jar
 hadoop-yarn-server-common-3.2.0.jar
 hadoop-yarn-server-web-proxy-3.2.0.jar
+hive-beeline-2.3.6.jar
+hive-cli-2.3.6.jar
+hive-common-2.3.6.jar
+hive-exec-2.3.6-core.jar
+hive-jdbc-2.3.6.jar
+hive-llap-common-2.3.6.jar
+hive-metastore-2.3.6.jar
+hive-serde-2.3.6.jar
+hive-service-rpc-2.3.6.jar
+hive-shims-0.23-2.3.6.jar
+hive-shims-2.3.6.jar
+hive-shims-common-2.3.6.jar
+hive-shims-scheduler-2.3.6.jar
```

Comment:
cc @wangyum .

Reply:
Thank you @AngersZhuuuu @dongjoon-hyun It is exactly match.